### PR TITLE
Add line pulling latest Docker image

### PR DESCRIPTION
**Pull request by MaiHameed**
#55

### CODE REVIEW

**Comment by MaiHameed** [comment]
The first pull is in the try what you'll build section, so I added the secondary pull line just in case the end user decides to skip the twyb section.

**Comment by gkwan-ibm** [comment]
hi @Charlotte-Holt, would you review README.adco line 174?

File: README.adoc [hunk 1]
```diff
@@ -168,6 +170,13 @@ so it can be used later by the `system` and `inventory` services:
 mvn -pl models install
 ```
 
+If you have not pulled the latest Docker image yet, do so by running the following command:
```
Comment (Charlotte-Holt) [inline README.adoc:173]: Suggested update for line 173: 
"If you don't have the latest Docker image, pull it by running the following command:"

Comment (MaiHameed) [reply README.adoc]: Fixed with latest commit, thanks @Charlotte-Holt!

**Comment by gkwan-ibm** [comment]
hi @Charlotte-Holt  would you confirm and approve Mai's change?

**Comment by NimG98** [comment]
Verified on qa site, LGTM

**Pull request merged**